.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 50
+  rebase-strategy: disabled
+- package-ecosystem: docker
+  directory: "/"
+  schedule:
+    interval: monthly
+  open-pull-requests-limit: 50
+  rebase-strategy: disabled
```

Comment:
Not a blocker: This project doesn't have a custom `Dockerfile` at the moment since we're pulling the base image directly via Compose, so this could be removed for now.

Reply:
@phlipper Ah, I thought so, it was late and I wasn't sure. Will remove. 🙇🏼